R/vaccine_parameters.R

```diff
@@ -21,19 +20,18 @@
 #' @export
 set_rtss_epi <- function(
   parameters,
-  start,
-  end,
-  coverage,
+  coverages,
+  timesteps,
   age,
   min_wait,
   boosters,
   booster_coverage,
   seasonal_boosters = FALSE
   ) {
-  stopifnot(length(start) == 1 && start > 1)
-  stopifnot(length(end) == 1 && end > start)
+  if(length(coverages) != length(timesteps)){
+    stop("coverages and timesteps must align")
+  }
   stopifnot(min_wait >= 0)
```

Comment:
Can we make into an `if` test. The current error is "argument "min_wait" is missing, with no default". 
Modify to something like "The argument min_wait is missing. If using set_rtss_epi with seasonal_boosters = FALSE  in isolation, set to 0. If using set_rtss_epi with seasonal_boosters = TRUE or in combination with set_mass_rtss, specify min_wait appropriately". Want to be clear that it can be set to 0 in the case of no seasonal vaccination.

Reply:
hmmmm, not 100% sure I agree, the description you give is in the help file for that argument which I think is where all of those specific details should live?

Reply:
Hm okay, it is not quite clear where this fits in if only set_rtss_epi is used. maybe the default could be 0